### PR TITLE
Ocp4 gm deployment

### PR DESCRIPTION
**Pull request by rahug20**
**Please provide enough information so that others can review your pull request:**

1. Explain the **details** for making this change. What existing problem does the pull request solve? If it resolves an existing issue, be sure to use a Github [keyword](https://help.github.com/en/articles/closing-issues-using-keywords) to automatically close it.

Made changes to `securityContext ` spec attribute in the charts. Upon testing , the `securityContext ` is same for OCP versions and kubernetes environments.  

2. What **changes to custom.yaml** are required?

N/A

3. Have you documented any additional setup steps or configurations options?

Yes, added comments

4. Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

Run

make install in OCP4 or any other kubernetes environment and cluster should be running successfully.

P.S you will see this error if you are using quickstart certs https://github.com/DecipherNow/helm-charts/issues/743

### CODE REVIEW

**Comment by moosecanswim** [comment]
Looks good to me!

**Comment by moosecanswim** [comment]
@zoemccormick do you mind checking this out? I've been out of the github helm charts for a bit so you might recognize something that i dont.

File: global.yaml [hunk 1]
```diff
@@ -37,7 +37,7 @@ global:
       name: waiter-sa
   # Global sidecar config (supports version and envvars)
   sidecar:
-    version: 1.4.4
+    version: 1.4.5
```
Comment (zoemccormick) [inline global.yaml:40]: should update edge version [here](https://github.com/DecipherNow/helm-charts/blob/209431505c361aad4726917d8d3d73f961c8ef94/edge/values.yaml#L37) too for consistency

**Review by zoemccormick — APPROVED** [review]
this looks good to me! tested in eks and k3d

**Comment by chrisbsmith** [comment]
@rahug20 you will need to forward-port this to the `release-2.3` branch also.

**Pull request merged**